local/Makefile

```diff
@@ -107,6 +110,36 @@ stop:  ## Stop the local Jenkins master and related processes
 			--file docker-compose.yml \
 			stop $${SERVICE}
 
+.PHONY: wiremock-bo
+wiremock-bo:  ## Hit the Blue Ocean URLs for wiremock tests
+	for JOB in $(WIREMOCK_JOBS); do\
+		for SECTION in $(WIREMOCK_SECTIONS); do\
+        	curl -I -s http://localhost:8080/blue/organizations/jenkins/it%2FgetBuildInfoJsonFiles%2F$$JOB/detail/$$JOB/1/$$SECTION/ > /dev/null ;\
+		done \
+	done
+
+	cp -R __files ../src/test/resources/
+	cp -R mappings ../src/test/resources/
```

Comment:
@v1v this is adding more files to git (https://github.com/elastic/apm-pipeline-library/pull/1279/files#diff-692753fe130202eab8a7b28ba88ef6d87756de0a3f3aa5fe8b33613c371f021cL42), but I'm not sure how to run the tests for the new files